release-notes/docs/3.10.0.md

```diff
@@ -0,0 +1,149 @@
+# 3.10.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Supporting remote first-time login](#supporting-remote-first-time-login)
+- [Supporting linked docs](#supporting-linked-docs)
+- [Privacy Policies](#privacy-policies)
+- [More granular replication depth configuration for offline users](#more-granular-replication-depth-configuration-for-offline-users)
+- [Improved geolocation data saved for reports](#improved-geolocation-data-saved-for-reports)
+- [Customizable header tabs icons](#customizable-header-tabs-icons)
+- [Improved performance of processing deleted docs](#improved-performance-of-processing-deleted-docs)
+- [And more...](#and-more)
+
+## Known issues
+
+None.
+
+## Upgrade notes
+
+Sentinel no longer advances its transition processing seq in `_local/sentinel-meta-data`. Instead, it advances the transition processing seq in `_local/transitions-seq`. Additionally, a new document `_local/background-seq` advances the deletion processing seq. 
```

Comment:
I'm not sure this should be mentioned, or at least, not mentioned here. It sounds like we require people to do something before upgrade but that's not the case. Maybe it's a breaking change?

Reply:
It kindof is a breaking change. Often, TLs and SREs need to advance Sentinel process seq, so I think we should mention this somewhere.

Reply:
Fair enough. Let's put it in the breaking changes section.